tests/README.md

```diff
@@ -102,27 +102,29 @@ bats <path/to/file.bats>
 
 Additionally tests can be filtered via tags using the `--filter-tags <tags,...>` argument.
```

Comment:
This looks odd when it's above the `cypress` instructions. Maybe add a sentence similar to the one above the `bats` instructions or move this?